store/models.py

```diff
@@ -30,3 +30,10 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+
+class BookRating(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, related_name='user',null=True,blank=True,on_delete=models.SET_NULL)
+    rating = models.FloatField(default=0.0)
```

Comment:
The rating shall be given as an integer - please read proper instructions.
The user should not be null here, and a better option would be to use `on_delete=models.CASCADE`

You could have also used `unique_together` META option here.